serverless.yml

```diff
@@ -0,0 +1,46 @@
+service: serverless-graphql-apollo
+
+frameworkVersion: ">=1.21.0 <2.0.0"
+
+provider:
+  name: aws
+  runtime: nodejs8.10
+  stage: dev
+  region: us-east-1
+  environment:
+    MONGODB_URL: ${env:MONGODB_URL}
+    GRAPHQL_ENDPOINT_URL: ${env:GRAPHQL_ENDPOINT_URL}
+  iamRoleStatements:
```

Comment:
This needs to be locked down more, again, let's get a thing out, and lock down after we have the functionality we need. This issue is helpful: https://github.com/serverless/serverless/issues/588 see the last comment.

Reply:
Raised https://github.com/freeCodeCamp/open-api/issues/41